scripts/scil_fibertube_tracking.py

```diff
@@ -0,0 +1,294 @@
+             '[%(default)s]')
+    track_g.add_argument(
+        '--rk_order', metavar="K", type=int, default=1,
+        choices=[1, 2, 4],
+        help="The order of the Runge-Kutta integration used \n"
+             'for the step function. \n'
+             'For more information, refer to the note in the \n'
+             'script description. [%(default)s]')
+    track_g.add_argument(
+        '--max_invalid_nb_points', metavar='MAX', type=int,
+        default=0,
+        help='Maximum number of steps without valid \n'
+             'direction, \nex: No fibertube intersecting the \n'
+             'tracking sphere or max angle is reached.\n'
+             'Default: 0, i.e. do not add points following '
+             'an invalid direction.')
+    track_g.add_argument(
+        '--forward_only', action='store_true',
+        help='If set, tracks in one direction only (forward) \n'
+             'given the \ninitial seed.')
+    track_g.add_argument(
+        '--keep_last_out_point', action='store_true',
+        help='If set, keep the last point (once out of the \n'
+             'tracking mask) of the streamline. Default: discard \n'
+             'them. This is the default in Dipy too. \n'
+             'Note that points obtained after an invalid direction \n'
+             '(based on the propagator\'s definition of invalid) \n'
+             'are never added.')
+
+    seed_group = p.add_argument_group(
+        'Seeding options',
+        'When no option is provided, uses --nb_seeds_per_fibertube 5.')
+    seed_group.add_argument(
+        '--nb_seeds_per_fibertube', type=int, default=5,
+        help='The number of seeds planted in the first segment \n'
+             'of each fibertube. The total amount of streamlines will \n'
+             'be [nb_seeds_per_fibertube] * [nb_fibertubes]. [%(default)s]')
+    seed_group.add_argument(
+        '--nb_fibertubes', type=int,
+        help='If set, the script will only track a specified \n'
+             'amount of fibers. Otherwise, the entire tractogram \n'
+             'will be tracked. The total amount of streamlines \n'
+             'will be [nb_seeds_per_fibertube] * [nb_fibertubes].')
+
+    rand_g = p.add_argument_group('Random options')
+    rand_g.add_argument(
+        '--rng_seed', type=int, default=0,
+        help='If set, all random values will be generated \n'
+        'using the specified seed. [%(default)s]')
+    rand_g.add_argument(
+        '--skip', type=int, default=0,
+        help="Skip the first N seeds. \n"
+             "Useful if you want to create new streamlines to "
+             "add to \na previously created tractogram with a "
+             "fixed --rng_seed.\nEx: If tractogram_1 was created "
+             "with -nt 1,000,000, \nyou can create tractogram_2 "
+             "with \n--skip 1,000,000.")
+
+    out_g = p.add_argument_group('Output options')
+    out_g.add_argument(
+        '--do_not_save_seeds', action='store_true',
```

Comment:
double negation is always tricky... why not use save_seeds ?

Reply:
Because seeds are pretty much required if you want to be able to score the tractogram. They should be saved by default. If anything, I can just remove then option to not save them.